spacify-portal/src/main/java/org/uci/spacifyPortal/controllers/UserController.java

```diff
@@ -127,6 +131,29 @@ public ResponseEntity<MessageResponse> updateUserProfile(@RequestBody UserDetail
         }
     }
 
+    // close account functionality. Erases user from the system.
```

Comment:
For deleting user, we may develop ui button for this later. Can remove it if you want